superset-frontend/plugins/legacy-preset-chart-deckgl/src/layers/Arc/Arc.tsx

```diff
@@ -98,7 +99,50 @@ export const getLayer: GetLayerType<ArcLayer> = function ({
       filterState,
       emitCrossFilters,
     }),
+    opacity: filterState?.value ? 0.1 : 1,
   });
 };
 
-export default createCategoricalDeckGLComponent(getLayer, getPoints);
+export const getHighlightLayer: GetLayerType<ArcLayer> = function ({
+  formData,
+  payload,
+  filterState,
+}) {
+  const fd = formData;
+  const data = payload.data.features;
+
+  const getColor = (d: {
+    sourcePosition: [number, number];
+    targetPosition: [number, number];
+  }) => {
+    const sourcePosition = filterState?.value[0];
+    const targetPosition = filterState?.value[1];
+
+    if (
+      sourcePosition &&
+      targetPosition &&
+      d.sourcePosition[0] === sourcePosition[0] &&
+      d.sourcePosition[1] === sourcePosition[1] &&
+      d.targetPosition[0] === targetPosition[0] &&
+      d.targetPosition[1] === targetPosition[1]
+    ) {
+      return [255, 0, 0, 255] as Color;
+    }
+
+    return [0, 0, 0, 0] as Color;
```

Comment:
We use [255, 0, 0, 255] and [0, 0, 0, 0] a lot in this PR. Could we replace them with a const? maybe sth like `const redColorArray` and `transparentColorArray`?